tests/test_sync.py

```diff
@@ -30,6 +32,22 @@ def test_get(server):
     assert response.status_code == 200
     assert response.content == b"Hello, world!"
     assert response.text == "Hello, world!"
+    assert response.protocol == "HTTP/1.1"
+    assert (
+        datetime.datetime.now(datetime.timezone.utc)
+        - email.utils.parsedate_to_datetime(response.headers["date"])
+    ) < datetime.timedelta(seconds=1)
+    expected_headers = httpcore.Headers(
+        {
+            "server": "uvicorn",
+            "content-type": "text/plain",
+            "transfer-encoding": "chunked",
+        }
+    )
```

Comment:
The header assertions here get a bit grungy - I'd suggest we just drop those. Tho adding the protocol line is a good idea.